proto-lens-protoc/src/Data/ProtoLens/Setup.hs

```diff
@@ -130,7 +135,12 @@ generatingSpecificProtos root getProtos hooks = hooks
     }
   where
     generateSources p l = do
-        files <- getProtos p
+        -- `makeAbsolute f` returns the absolute path of f as though it were
+        -- specified relative to root.
+        let makeAbsolute f = if isRelative f
```

Comment:
I believe you can replace `makeAbsolute` with just  `</>`, which already does nothing if `f` is absolute:
https://github.com/ghc/packages-filepath/blob/master/System/FilePath/Internal.hs#L644